app/controllers/imports_controller.rb

```diff
@@ -1,71 +1,45 @@
 class ImportsController < ApplicationController
   before_action :set_podcast
-  before_action :set_import, only: %i[show edit update destroy]
 
   # GET /imports
   def index
-    @imports = [] # Import.all
-  end
-
-  # GET /imports/1
-  def show
-  end
-
-  # GET /imports/new
-  def new
-    # @import = Import.new
-  end
-
-  # GET /imports/1/edit
-  def edit
+    @imports = PodcastImport.where(podcast_id: @podcast).order(created_at: :desc)
```

Comment:
decided on utilizing the `index` action for both "show" and "new", where a user can both see all of the previous imports, as well as start a new one.